OpenRA.Game/GameRules/WeaponInfo.cs

```diff
@@ -42,6 +42,8 @@ public sealed class WeaponInfo
 		[Desc("The sound played when the weapon is fired.")]
 		public readonly string[] Report = null;
 
+		public readonly bool OnlyPlayReportOnFirstBurst = false;
```

Comment:
I assume that's meant to be in addition to the existing `Report`?

In my opinion we should at least call it `FirstBurstReport`, as I'm sure not every modder will read the Desc and just `BurstReport` is a bit ambigous (if I was a 3rd-party modder my first assumption would be that it plays for each burst and wonder where's the difference to the normal `Report`.

Also, should this play *in addition to* or *instead of* `Report`?